spec/requests/registrations_spec.rb

```diff
@@ -1313,3 +1343,21 @@ def refund_response(capture_id)
     ],
   }.to_json
 end
+
+def payment_confirmation_webhook(intent)
+  {
+    id: "evt_3P6aXQJzvpX2joEA18jzmlxq",
+    object: "event",
+    account: "acct_1FcgXcJzvpX2joEA",
```

Comment:
This can (and should!) be read from the actual Stripe account ID that was also used to make the requests creating and confirming the PI

Reply:
Changed

Reply:
(and refactored into a variable for this test as the account ID is called elsewhere)